project1/backend/models/resolutionModel.js

```diff
@@ -0,0 +1,86 @@
+const AppError = require('../utils/AppError');
+const formatName = require('../utils/formatName.js');
+const Ajv = require('ajv');
+
+const ajv = new Ajv();
+
+class ResolutionModel {
+  constructor() {
+    this.patients = [];
+    this.patientSchema = {
+      type: 'object',
+      properties: {
+        name: { type: 'string', minLength: 2, maxLength: 20 },
+        resolution: { type: 'string', maxLength: 400 },
+        expiry: { type: 'integer' },
+      },
+      required: ['name', 'resolution', 'expiry'],
+      additionalProperties: false,
+    };
+  }
+
+  create(body) {
+    this.filterTTL();
+
+    // <--- MOVE TO A SEPARATE DTO SECTION ?
+
+    // validation start
+
+    const validateEnqueueSchema = ajv.compile(this.patientSchema);
```

Comment:
Can we create validation middleware for this? 
Examples: https://express-validator.github.io/docs/, https://dev.to/simonplend/learn-how-to-handle-validation-in-express-3blb